app/src/renderer/components/transactions/LiDistrTransaction.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+  <li-transaction
+    :color="`#F2B134`"
+    :time="transaction.time"
+    :block="transaction.height"
+  >
+    <template v-if="txType === `cosmos-sdk/MsgWithdrawDelegationReward`">
+      <div slot="caption">
+        Withdraw rewards
+      </div>
+      <div slot="details">
+        From&nbsp;<router-link :to="url + '/' + tx.validator_addr">
+          {{
+            moniker(tx.validator_addr)
+          }}
+        </router-link>
+      </div>
+    </template>
+    <template v-else-if="txType === `cosmos-sdk/MsgSetWithdrawAddress`">
+      <div slot="caption">
+        Update withdraw address
+      </div>
+      <div slot="details">
+        To {{ tx.withdraw_address }}
+      </div>
+    </template>
+    <template v-else-if="txType === `cosmos-sdk/MsgWithdrawValidatorCommission`">
+      <div slot="caption">
+        Withdraw validator commission
+      </div>
+      <div slot="details">
+        From&nbsp;<router-link :to="url + '/' + tx.validator_address">
+          {{
+            moniker(tx.validator_address)
+          }}
+        </router-link>
+      </div>
+    </template>
+  </li-transaction>
+</template>
+
+<script>
+import LiTransaction from "./LiTransaction"
+import { pretty, atoms } from "../../scripts/num.js"
+
+export default {
+  name: `li-distr-transaction`,
```

Comment:
please, use the full words

Reply:
what do you mean ?

Reply:
`li-distribution-transaction`

short forms / abbreviations leave room for confusion and inconsistency